include/deal.II/grid/tria.h

```diff
@@ -949,7 +949,7 @@ namespace internal
  * been called.
  *
  *
- * <h3>%Boundary approximation</h3>
+ * <h3>Boundary approximation</h3>
```

Comment:
This is actually on purpose: it escapes what would otherwise be a link to the `Boundary` class.